crates/bevy_app/src/lib.rs

```diff
@@ -35,7 +35,7 @@ use bevy_ecs::schedule::StageLabel;
 #[derive(Debug, Hash, PartialEq, Eq, Clone, StageLabel)]
 pub enum CoreStage {
     /// The [`Stage`](bevy_ecs::schedule::Stage) that runs before all other app stages.
```

Comment:
Needs more doc comments.

Reply:
Agreed. Damn you're quick. :sweat_smile: